client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -202,6 +206,49 @@ class PurchasesListing extends Component {
 		);
 	}
 
+	getProductActionButtons( purchase ) {
+		const { translate, selectedSiteSlug: site } = this.props;
+		const actionButton = this.getActionButton( purchase );
+
+		let serviceButton = null;
+		if ( isJetpackBackup( purchase ) ) {
+			serviceButton = (
+				<Button
+					href={ addQueryArgs(
+						{ site, source: 'jetpack-backup-dash' },
+						'https://jetpack.com/redirect/'
+					) }
+					compact
+				>
+					{ translate( 'View Backups' ) }
+					&nbsp;
+					<Gridicon icon="external" />
+				</Button>
+			);
+		} else if ( isJetpackScan( purchase ) ) {
+			serviceButton = (
+				<Button
+					href={ addQueryArgs(
+						{ site, source: 'jetpack-scan-dash' },
```

Comment:
this should be calypso-scanner instead of jetpack-scan-dash

Reply:
Resolved in 3f770fa.